index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        
+        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,400i|Nunito:300,300i" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/png" href="img/favicon.png">
+        
+        <title></title>
+    </head>
+
+    <body>
+        <main class="Main-container">
+            <header class="Header">
+                <nav class="container">
+                    <div class="logoplaceholder">
+                        <h1> Welcome</h1>
+                    </div>
+                    <div class="menu__button">
+                        <div class="Menu__Icon">
+                            <a href="#" class="menu__link">
+                                <img src="img/Icon - Menu.svg" alt="Menu Icon">
+                            </a>
+                        </div>
+                    </div>
+    
+                </nav>
+            </header>
+            <section class="Headline">
+                <div class="Headline__primary">
+                   <h1 class="Text__intro"> <em>Hey there. I’m Solomon <br> I’m a software developer</em></h1> 
```

Comment:
- [ ] As in the Figma design, this text should not be _italic_. Please pay attention also to use the same colors as in the Figma template. You can click on a figma element and see its values. 😄